### PR TITLE
chore(Pre releases): prepare scripts for pre releases

### LINKED ISSUE #1670: Update all assembly versions when releasing new Stryker package
We should probably add the updating of this version number to the prepare-release.js script we use to bump the version number in the other assemblies.

_Originally posted by @rouke-broersma in https://github.com/stryker-mutator/stryker-net/pull/1664#discussion_r693360626_

### PR DESCRIPTION
**Pull request by richardwerkman**
Fix #1670 
Is part of #1642

### CODE REVIEW

File: prepare-release.js [hunk 1]
```diff
@@ -20,39 +20,51 @@ const packages = [
...
+
         console.log(`Updating changelog for ${package.name}`);
         exec(`npx conventional-changelog-cli -p angular --infile "${package.path}/CHANGELOG.md" --same-file --commit-path ${package.path} --tag-prefix "${package.name}@"`);
         commitMessageLines.push(`- ${package.name}@${newVersionNumber}`);
     });
 
-    console.log('Updating azure-pipelines.yml');
-    replaceVersionNumber('./azure-pipelines.yml', `VersionBuildNumber: $[counter('${oldVersionNumber}', 1)]`, `VersionBuildNumber: $[counter('${newVersionNumber}', 1)]`);
-    replaceVersionNumber('./azure-pipelines.yml', `PackageVersion: '${oldVersionNumber}'`, `PackageVersion: '${newVersionNumber}'`);
+    // console.log('Updating azure-pipelines.yml');
```
Comment (rouke-broersma) [inline prepare-release.js:55]: Should this be commented out

Comment (richardwerkman) [reply prepare-release.js]: Nope, fixed

**Comment by rouke-broersma** [comment]
It's probably a good idea to change all the project versions to v1.0.0 (for example regex mutator project)

**Comment by richardwerkman** [comment]
@rouke-broersma the regex mutator project doesn't have a version. It's just included in stryker.core and will get it's version from there. If we have plans to release it as a seperate package I see the benefit but for now I don't think it needs a version.

**Comment by rouke-broersma** [comment]
> @rouke-broersma the regex mutator project doesn't have a version. It's just included in stryker.core and will get it's version from there. If we have plans to release it as a seperate package I see the benefit but for now I don't think it needs a version.

My bad I thought versions were mandatory. the DataCollector does have a version, I think it probably does need a version there: https://github.com/stryker-mutator/stryker-net/blob/master/src/Stryker.DataCollector/Stryker.DataCollector/Stryker.DataCollector.csproj

**Pull request merged**